### DISCUSSION TITLE
请问doris用jemeter做insert的压测，一分钟200并发，fe就会中途挂掉，为什么呢？

### DISCUSSION THREAD

**Discussion by Tongyaoyu** [apache/doris#19965] [A - General / Q&A]
我用docker起了fe和be，doris是1.2.2版本，然后使用jmeter做压测，只有每分钟200的并发，每次中途钱100来条成功，后面全部失败，fe就会直接挂掉，也设置了max_user_connections为10000，为什么这一点并发fe就没了呢？
![image](https://github.com/apache/doris/assets/53547222/cbf4db7d-9954-4d0c-9dbc-aac5b85f9048)

**Comment by LemonLiTree** [comment]
insert into value？or insert into select？
